### PR TITLE
Anup_Assignment_31_Done_20mins

### PR DESCRIPTION
**Pull request by AnupSJun20**
Please review.
Program1

### CODE REVIEW

File: src/anupSJun20/SumOfThreeNumbersTryCatch.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class SumOfThreeNumbersTryCatch {
+	void findSumofThreeNumbers() {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Please enter the number of digits to add :");
+		int totalcount = sc.nextInt();
```
Comment (harshvegada) [inline src/anupSJun20/SumOfThreeNumbersTryCatch.java:21]: What if user gives String values in int
How I will handle this scenario

File: src/anupSJun20/SumOfThreeNumbersTryCatch.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		int count = 0;
+		int sum = 0;
+		int[] input = new int[totalcount];
+		
+		while(count<totalcount)
+			try {
+				System.out.println("Please enter the number :" + (count+1));
+				input[count] = sc.nextInt();
```
Comment (harshvegada) [inline src/anupSJun20/SumOfThreeNumbersTryCatch.java:29]: Without using array, u can add values

**Review by harshvegada — APPROVED** [review]
No Major changes required
Code looks Good to me
Keep it up 👍

File: src/anupSJun20/SumOfThreeNumbersTryCatch.java (hunk 1 above)
Comment (KrishnaKTechnocredits) [reply src/anupSJun20/SumOfThreeNumbersTryCatch.java:21]: You should handle this line in try catch.

**Pull request merged**